src/models/CheckoutParams.ts

```diff
@@ -0,0 +1,29 @@
+import AmountMoney from "./AmountMoney";
+import TipSettings from "./TipSettings";
+
+interface CheckoutParams {
+    amountMoney: AmountMoney;
+
+    expirationYear?: number;
```

Comment:
expirationYear, lastFourDigits and postalCode doesn't seem to belong here either.

Reply:
Removed these properties.